visualization/app/codeCharta/ui/blacklistPanel/blacklistPanel.component.ts

```diff
@@ -23,6 +25,15 @@ export class BlacklistPanelController implements BlacklistSubscriber {
 		const excluded: BlacklistItem[] = []
 
 		for (const item of blacklist) {
```

Comment:
Should we add a small comment to explain what that algorithm does?
I understand it that way: "Cut enough folders of the path from the beginning so that the display_name is short enough.

Reply:
I have another idea for a possible improvement.
We could add the original file path as a `title="{{path}}"` attribute, so that the full path is still visible if you hover the shortened name.

Reply:
Removed previous comment. A title attribute is already implemented.